arch/arm64/src/zynq-mpsoc/Kconfig

```diff
@@ -35,6 +35,136 @@ config ZYNQ_MPSOC_UART1
 	---help---
 		Select to enable support for UART1.
 
+config ZYNQ_ENET
+	bool "Ethernet"
+	default n
+	select ARCH_HAVE_PHY
+	select ARCH_HAVE_NETDEV_STATISTICS
+
 endmenu # XILINX ZYNQ_MPSOC Peripheral Selection
 
+menu "Ethernet Configuration"
+	depends on ZYNQ_ENET
+
+choice
+	prompt "ZYNQ Ethernet Interface"
+	default ZYNQ_ENET4
```

Comment:
Why does ENET4 is default instead of ENET1?

Reply:
Because GMAC3 belong to PS of ZYNQ MPSOC, GMAC0~GMAC2 may depends on PL of ZYNQ MPSOC

Reply:
whats more, ZCU111 board use GMAC3